### PR TITLE
Parse new concentration field

### PR DESCRIPTION
**Pull request by islean**
## Description

Part of the new order form was to add a new field where the customer can set concentration on sample level in ng/ul. This is intended to be used when considering a bypass of the quality control. This PR makes sure that we parse this value.

### Changed

- Added field to hold new concentration field.

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/models/orders/excel_sample.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ class ExcelSample(OrderSample):
     concentration: Annotated[str, AfterValidator(numeric_value)] = Field(
         None, alias=ExcelSampleAliases.CONCENTRATION
     )
+    concentration_ng_ul: str = Field(None, alias=ExcelSampleAliases.CONCENTRATION_NG_UL)
```
Comment (diitaz93) [inline cg/models/orders/excel_sample.py:32]:
```suggestion
    concentration_ng_ul: str | None = Field(None, alias=ExcelSampleAliases.CONCENTRATION_NG_UL)
```

**Review by diitaz93 — APPROVED** [review]
👍

File: cg/models/orders/excel_sample.py (hunk 1 above)
Comment (islean) [reply cg/models/orders/excel_sample.py:32]: And also, the field is marked as optional in the base class OrderSample, which also seems to handle this for cases where the user does not themselves give None as input.

**Review by henrikstranneheim — APPROVED** [review]
👍

**Pull request merged**